lib/react_client.dart

```diff
@@ -510,6 +511,28 @@ _convertBoundValues(Map args) {
 /// to the original Dart functions (the input of [_convertEventHandlers]).
 final Expando<Function> _originalEventHandlers = new Expando();
 
+/// Returns the props for a [ReactElement] or composite [ReactComponent] [instance],
+/// shallow-converted to a Dart Map for convenience.
```

Comment:
#nit might be nice to mention the event handler unconverting

Reply:
Added